web-ui/src/pages/TeamSkillReportPage.jsx

```diff
@@ -46,11 +47,11 @@ const TeamSkillReportPage = () => {
     }
     if (memberSkillsFound && memberProfiles) {
       setAllSearchResults(memberSkillsFound);
-      setSearchResults(
-        memberSkillsFound.filter(mSkill =>
-          selectedMembers.some(member => member.id === mSkill.id)
-        )
+      let membersSelected = memberSkillsFound.filter(mSkill =>
```

Comment:
I think `membersSelected` is going to be an array that contains a subset of the elements in `memberSkillsFound`. Is that what you want? It seems like you many want an array of member objects rather than skill objects?